.travis.yml

```diff
@@ -8,6 +8,9 @@ python:
   - "3.5"
   - "3.6"
 before_install:
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+        pip install pep8;
```

Comment:
Hi @nishnik,
It has been renamed to [pycodestyle](https://github.com/PyCQA/pycodestyle) ! :)